features/jest/client/src/main/java/org/opennms/plugins/elasticsearch/rest/template/DefaultTemplateLoader.java

```diff
@@ -59,6 +62,16 @@ public String load(Version serverVersion, String resource) throws IOException {
         throw new NullPointerException(String.format("No template found for server version %s and resource %s.",
                 serverVersion, resource));
     }
+    
+    protected String getTemplate(String resource) throws IOException {
```

Comment:
Using exceptions to dictate control flow is typically to be avoided. However the class you are relying on `ConfigFileConstants` does not seem to offer a convenient alternative.

I created a commit that updates the class to provide one. If you can take that commit and add it to your PR we can clean this up a bit and have a better option going forward. Please find the commit here: https://github.com/OpenNMS/opennms/commit/13a2c9920231f34e6501bce2f765df10b4254b80

Please let me know if you have any questions with how to do that.

Once you have that commit you can replace the method content for `getTemplate` with the following:
```
Optional<Path> configPath = ConfigFileConstants.getConfigFilePathByName(resource);
if(configPath.isPresent()) {
  return new String(Files.readAllBytes(configPath.get()));
}
return getResource(resource);
```